### ISSUE TITLE
image handling

### ISSUE THREAD

**Issue by WolfgangFahl** [samwilson/diagrams-extension#9]
```
<graphviz>
digraph tools  {
  graphviz [  image="GraphVizLogo.png" ]
}
</graphviz>
```

should work see http://wiki.bitplan.com/index.php/GraphViz#Images

**Comment by WolfgangFahl** [comment]
see https://github.com/wikimedia/mediawiki-extensions-GraphViz/blob/master/includes/GraphViz.php#L860

**Comment by WolfgangFahl** [comment]
see also https://gitlab.com/graphviz/graphviz/issues/1664 which would simplify this a lot

**Comment by samwilson** [comment]
This should be fixed in 0.6.0.
